ai/anthropic/v0/config/tasks.json

```diff
@@ -0,0 +1,298 @@
+        },
+        "role": {
+          "description": "The message role, i.e. 'system', 'user' or 'assistant'",
+          "instillFormat": "string",
+          "instillUIOrder": 0,
+          "title": "Role",
+          "type": "string"
+        }
+      },
+      "required": [
+        "role",
+        "content"
+      ],
+      "title": "Chat Message",
+      "type": "object"
+    },
+    "common": {
+      "description": "Input",
+      "instillEditOnNodeFields": [
+        "image_base64",
+        "model_name"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "image_base64": {
+          "description": "Image base64",
+          "instillAcceptFormats": [
+            "image/*"
+          ],
+          "instillUIOrder": 2,
+          "instillUpstreamTypes": [
+            "reference"
+          ],
+          "title": "Image",
+          "type": "string"
+        },
+        "model_name": {
+          "description": "The Instill Model model to be used.",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUIOrder": 0,
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "title": "Model Name",
+          "type": "string"
+        }
+      },
+      "required": [
+        "image_base64",
+        "model_name"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "extra_params": {
+      "description": "Extra Parameters",
+      "instillAcceptFormats": [
+        "semi-structured/object"
+      ],
+      "instillUIOrder": 3,
+      "instillUpstreamTypes": [
+        "reference"
+      ],
+      "required": [],
+      "title": "Extra Parameters",
+      "type": "object"
+    }
+  },
+  "TASK_TEXT_GENERATION_CHAT": {
+    "instillShortDescription": "Provide text outputs in response to text inputs.",
+    "description": "Anthropic's text generation models (often called generative pre-trained transformers or large language models) have been trained to understand natural language, code, and images. The models provide text outputs in response to their inputs. The inputs to these models are also referred to as \"prompts\". Designing a prompt is essentially how you \u201cprogram\u201d a large language model model, usually by providing instructions or some examples of how to successfully complete a task.",
+    "input": {
+      "description": "Input",
+      "instillEditOnNodeFields": [
+        "prompt",
+        "model_name"
+      ],
+      "instillUIOrder": 0,
+      "properties": {
+        "chat_history": {
+          "description": "Incorporate external chat history, specifically previous messages within the conversation. Please note that System Message will be ignored and will not have any effect when this field is populated. Each message should adhere to the format: : {\"role\": \"The message role, i.e. 'system', 'user' or 'assistant'\", \"content\": \"message content\"}.",
```

Comment:
Important
I remember the description has the dependency of Console.
It means this long description would damage UI in the Console.
Please help us check if it makes sense in frontend.

Same as other descriptions

Reply:
It look fine to me.
![Screenshot from 2024-06-14 12-35-22](https://github.com/instill-ai/component/assets/54308739/a0c8b570-1d04-4421-b890-9eeeaf0a2e14)